README.md

```diff
@@ -541,6 +542,29 @@ end
 
 [grape-swagger]: https://github.com/ruby-grape/grape-swagger
 
+## Configuration
+
+Grape counts with a module `Grape::Config` for some basic configuration at load time.
+Currently the configurable settings are:
+
+* `param_builder`: Sets the default [Parameter Builder](#parameters), by default: `Grape::Extensions::ActiveSupport::HashWithIndifferentAccess::ParamBuilder`
+
+To change a setting value make sure that at some point on load time the code the following code runs
+
+```ruby
+Grape::Config.configure do |config|
```

Comment:
It should be `Grape.configure do |config|`, because we're not configuring the configuration, we're configuring Grape.